addons/wardrobe/BaseClasses.hpp

```diff
@@ -0,0 +1,94 @@
+#define CN_SOUND(base,dur) QUOTE(TRIPLES(ADDON,base,dur))
+
+class GVAR(base) {
+
+    class modifiableTo {        
+    };
+
+    // describes the components of the current item
+    // When the current wearable is being changed into something, that does not have these 
+    components[] = {};
+
+    // Supports Multiple Sounds, will pick one by random
+    sound[] = {CN_SOUND(fabric,06),CN_SOUND(fabric,07),CN_SOUND(fabric,16),CN_SOUND(fabric,20),CN_SOUND(fabric,25)};
+    sound_timing = 0;    // [0..1] 0 at the start of the action, 0.5 half way during the duration of the action - always, if completed or not. 1 at the end, only when completed
+
+    // Gesture
+    gesture = "Gear";
+
+    // These will be read from the target class, so for example, the uniformclass with the rolled up sleaves, it should say "Roll Up Sleeves"
+    alternativePicture = "";
+    alternativeDisplayName = "";
+
+    duration = 1; // Minimum value: 1 - Anything above will produce a progressbar
+
+    fallbackComponent = ""; // To be used as an alternative for components where the intended component is from another addon/mod in case the source addon is not loaded
+};
+
+// Common base class for uniforms with sleeves up/down variants
+class GVAR(base_U_sleeves_up): GVAR(base) {
+    alternativeDisplayName = CSTRING(sleevesUp);  
+};
+class GVAR(base_U_sleeves_down): GVAR(base) {
+    alternativeDisplayName = CSTRING(sleevesDown);
+};
+
+// Common base class for uniforms with gloves on/off variants
+class GVAR(base_U_gloves_on): GVAR(base) {
+    alternativeDisplayName = CSTRING(glovesOn); 
+};
+class GVAR(base_U_gloves_off): GVAR(base) {
+    alternativeDisplayName = CSTRING(glovesOff);
+};
+
+// Common base class for uniforms who are open/closed in the front
+class GVAR(base_U_jacket_open): GVAR(base) {
+    alternativeDisplayName = CSTRING(jacketOpen); 
+};
+class GVAR(base_U_jacket_closed): GVAR(base) {
+    alternativeDisplayName = CSTRING(jacketClose);
+};
+
+// Common base class for helmets with a visor that can be flipped up or down
+class GVAR(base_H_visor_up): GVAR(base) {
+    duration = 0.25;
+    gesture ="gestureNod";
```

Comment:
Add space wherever necessary:
```suggestion
    gesture = "gestureNod";
```

Reply:
https://github.com/OverlordZorn/ACE3/commit/c003901077b44019aa6d18d6b4e185fa72c88a07